tests/Unit/ActionTesting.hpp

```diff
@@ -1426,6 +1426,30 @@ class MockRuntimeSystem {
   }
   // @}
 
+  // @{
+  /// Invoke the simple action `Action` on the `Component` labeled by
+  /// `array_index` immediately.
+  template <typename Component, typename Action, typename Arg0,
+            typename... Args>
+  void threaded_action(const typename Component::array_index& array_index,
+                       Arg0&& arg0, Args&&... args) noexcept {
+    algorithms<Component>()
+        .at(array_index)
+        .template threaded_action<Action>(
+            std::make_tuple(std::forward<Arg0>(arg0),
+                            std::forward<Args>(args)...),
+            true);
+  }
+
+  template <typename Component, typename Action>
+  void threaded_action(
```

Comment:
Nope, I'm happy to drop it if you want a test.

Reply:
No I want this function, but since nothing is calling it right now I don't know if it works 😄 In particular, do I have to create my own node lock and pass it to the action as an argument or is one created automatically, like for `Parallel::threaded_action`?